README.rst

```diff
@@ -95,6 +97,24 @@ done the work for you. The images are uploaded to `quay.io`_ and are tagged
 for repeatable builds.
 
 
+manylinux_2_34 (AlmaLinux 9 based)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Toolchain: GCC 13
```

Comment:
If I understood https://github.com/pypa/manylinux/issues/1585#issuecomment-2198069688 correctly, the request from @mayeut is to use the new GCC 14 toolset for `manylinux_2_34`.